README.md

```diff
@@ -3,104 +3,83 @@
-4. Provide API to access the [DAG chain](https://www.notion.so/powerloom/PowerLoom-Protocol-Overview-c3bf9dd9323541118d46a4d8684565d1?pvs=4#c008da3137294ad6b065cfe0fcc05032). of snapshots
-5. Perform various high order operations such as time based indexing and aggregations on top of generated DAG chains
+<!-- TODO: add links to while paper for each of the functions below -->
+1. Store snapshots on decentralized storage protocols like IPFS and Filecoin.
+2. Submit snapshots to the on-chain source for consensus.
+3. Generate snapshotter report for snapshot submissions.
+4. Keep a local cache of finalized snapshots.
+5. Prune older snapshots.
 
-For a more detailed overview of the audit-protocol, please refer to the [protocol overview](https://powerloom.notion.site/PowerLoom-Protocol-Overview-c3bf9dd9323541118d46a4d8684565d1)
 
 ## Requirements
-* macOS or Linux (We're still working on windows support)
-* Python 3.8 or above and pip
+* macOS or Linux (We're still working on Windows support)
+* [Go 1.18](https://golang.org/doc/install) or above
 * [Redis](https://redis.io/docs/getting-started/installation/)
 * [RabbitMQ](https://www.rabbitmq.com/download.html)
 * [Pm2](https://pm2.keymetrics.io/docs/usage/quick-start/)
 * [IPFS](https://ipfs.tech/)
 
 ## Setup Instructions
-There are 2 ways to run audit-protocol, using docker or running processes directly in your environment.
-For simplicity it is recommended to run using docker.
+There are 2 ways to run Audit-protocol, using docker or running processes directly in your local environment.
+For simplicity, it is recommended to run using docker.
 
-
-### Steps using docker
-* Install [Docker Desktop](https://www.docker.com/products/docker-desktop/)
-* Follow the instructions below to setup virtual env and configure settings.
-* Run services via docker as mentioned [here](https://github.com/PowerLoom/deploy/tree/main#instructions-for-code-contributors)
-
-
-### Steps to run Directly
-Note that these steps will only run audit-protocol. If you want to run the complete system, follow the steps mentioned in [Docker Setup](#steps-using-docker)
-* Install required software as mentioned in [requirements](#requirements) section
-* Follow instructions to [setup virtual env](#setup-virtual-env) and [configure settings](#configure-settings)
-* Start all processes by running the command
+###  Configure settings
+* Copy `settings.example.json` present in root directory of the project to `settings.json` file, and populate it with the required values.
 
 ```shell
-pm2 start pm2.config.js
+cp settings.example.json settings.json
 ```
 
-### Setup Virtual Env
-* Setup a python virtual environment [Ref](https://docs.python.org/3/library/venv.html)
+- Configure the snapshotter account address (EVM compatible 0x...) in `settings.instance_id`
+- Configure the `pooler-namespace`
+- Configure `anchor_chain_rpc_url` in `settings.anchor_chain_rpc_url`
+- Configure the local cache directory path in `settings.local_cache_path`
```

Comment:
People don't need to configure this as this is fixed atm to `home/ubuntu/local_cache`